openshift/templates/tekton/pipelines/subscriber.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
We already have a `buildah-subscriber` pipeline.  What are you hoping to improve with this pipeline?  Additionally, we have a `buildah-main` pipeline that support automation through triggers that will build and deploy both the editor and subscriber when required.  

The file that controls the trigger is `/openshift/kustomize/tekton/triggers/listener.yaml`.  It looks like it may be missing the `added` check for subscriber content.  I'm not sure why it doesn't run most of the time though.

You can test all these things by temporarily creating a copy of all of these objects and running them against your own fork.